File: Frontend/src/app/Model/LaminationVendor.ts
export interface LaminationVendor {
  id: number | null | undefined;
  timeStamp: Date | string | null | undefined;
  name: string | null | undefined;
  createdBy: string | null | undefined;
  process: string | null | undefined;
  type: string | null | undefined;
  rate: number | null | undefined;
  status: boolean | null | undefined;
}

File: src/main/java/com/PrintLab/model/LaminationVendor.java
package com.PrintLab.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
public class LaminationVendor {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private LocalDateTime timeStamp;
    private String name;
    private String createdBy;
    private String process;
    private String type;
    private Double rate;
    private boolean status;

}
